### ISSUE TITLE
AgencyMemberInsert and FamilyMemberInsert issue

### ISSUE THREAD

**Issue by jrands** [Capping-CPCA/Database#75]
Getting the same issue on both these functions. Seems as though that when a new person is getting created they're not properly retrieving the personID and it's attempting to enter a null value into the PK of AgencyMember/Family Member

error is as follows: 
```
ERROR:  null value in column "familymemberid" violates not-null constraint
DETAIL:  Failing row contains (null, null, null, null, null).
CONTEXT:  SQL statement "INSERT INTO FamilyMembers(familyMemberID, relationship, dateOfBirth, race, sex) VALUES (fmID, rel, dob, rac, gender)"
PL/pgSQL function createfamilymember(text,text,character varying,relationship,date,race,sex,boolean,text,text,integer) line 9 at SQL statement
```

**Comment by jrands** [comment]
actually might have fixed it, can someone verify a50aa3aeddd1e4ba14ec2b0ab12cf0cb28d5c4e3

**Comment by CBadame** [comment]
Nice, looks like that fixed it.
